messages/messages.md

```diff
@@ -1,3 +1,19 @@
+# error.error
```

Comment:
Why the duplicated wording for this, `warning.warning`, and `info.info`?
I'd rather just do `error`, `warning`, and `info`.

Reply:
Just applying the grouping we do for messages related to errors and passing that along to warning and info.

Reply:
Ok let's name these `error.prefix`, `warning.prefix`, and `info.prefix` then so it's more descriptive instead of redundant.